From abd8b4e7daad4b7d7f54857e6847855a5b813e61 Mon Sep 17 00:00:00 2001
From: Thomas Epperson <tepperson@briskheat.com>
Date: Mon, 10 Sep 2018 17:34:58 -0400
Subject: [PATCH] Tweaks

---
 Makefile.am  | 2 --
 configure.ac | 3 +++
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 4bf9bc2..c85542b 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,4 @@
 bin_PROGRAMS = aws-prog1
 
-aws_prog1_CPPFLAGS = "$(CPPFLAGS) -I$(srcdir)/src"
-aws_prog1_CFLAGS = "$(CFLAGS) -I$(srcdir)/src"
 aws_prog1_SOURCES = src/main.c
 
diff --git a/configure.ac b/configure.ac
index 5c3921e..ad3f0f6 100644
--- a/configure.ac
+++ b/configure.ac
@@ -26,5 +26,8 @@ AC_CHECK_LIB([AwsIotSdk],[main],[], [
 ])
 
 AC_PROG_CC
+AC_PROG_INSTALL
+AC_PROG_MAKE_SET
+
 AC_CONFIG_FILES([Makefile])
 AC_OUTPUT
-- 
1.9.1

