From 439e7e12dcb7c95b2aebc6b5f36c7ced8a7cf944 Mon Sep 17 00:00:00 2001
From: Thomas Epperson <tepperson@briskheat.com>
Date: Mon, 30 Oct 2017 16:03:50 -0400
Subject: [PATCH] Fix for linux 4.13

---
 pwm.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pwm.c b/pwm.c
index f2be64f..58d2755 100644
--- a/pwm.c
+++ b/pwm.c
@@ -40,6 +40,7 @@
 #include <plat/dmtimer.h>
 #include <linux/kthread.h>
 #include <linux/delay.h>
+#include <linux/uaccess.h>
 
 #include "pwm.h"
 #include "pwm_ioctl.h"
@@ -332,7 +333,7 @@ static int pwm_timer_init(void)
 			pwm_dev[i].irq = omap_dm_timer_get_irq(pwm_dev[i].timer);
 
 			if (request_irq(pwm_dev[i].irq, match_handler,
-					IRQF_DISABLED | IRQF_SHARED,
+					IRQF_SHARED,
 					"pwm-match",
 					&pwm_dev[i])) {
 				printk(KERN_ERR
-- 
1.9.1

