From 39f8efcf6f8cc04816439ca9ea44e47ffb2e4d0f Mon Sep 17 00:00:00 2001
From: Thomas Epperson <tepperson@briskheat.com>
Date: Fri, 20 Apr 2018 11:14:34 -0400
Subject: [PATCH] Fix compilation

---
 pwm.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pwm.c b/pwm.c
index 3f10e4f..58d2755 100644
--- a/pwm.c
+++ b/pwm.c
@@ -40,6 +40,7 @@
 #include <plat/dmtimer.h>
 #include <linux/kthread.h>
 #include <linux/delay.h>
+#include <linux/uaccess.h>
 
 #include "pwm.h"
 #include "pwm_ioctl.h"
-- 
1.9.1

