From 8d415735a18aadefb30771e5b9c183ebf25d48ad Mon Sep 17 00:00:00 2001
From: Thomas Epperson <tepperson@briskheat.com>
Date: Mon, 13 Aug 2018 08:33:26 -0400
Subject: [PATCH] Increase maximum number of sectors that the spl will load.

---
 include/configs/socfpga_common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/socfpga_common.h b/include/configs/socfpga_common.h
index 0501bd1..30f2244 100644
--- a/include/configs/socfpga_common.h
+++ b/include/configs/socfpga_common.h
@@ -367,7 +367,7 @@ unsigned int cm_get_qspi_controller_clk_hz(void);
 #else
 #define CONFIG_SYS_MMCSD_FS_BOOT_PARTITION	3
 #define CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR	0xa00 /* offset 2560 sect (1M+256k) */
-#define CONFIG_SYS_U_BOOT_MAX_SIZE_SECTORS	800 /* 400 KB */
+#define CONFIG_SYS_U_BOOT_MAX_SIZE_SECTORS	900 /* 400 KB */
 #endif
 #endif
 
-- 
1.9.1

